akd_mysql/src/mysql.rs

```diff
@@ -740,6 +740,15 @@ impl Storage for AsyncMySqlDatabase {
 
         // this retrieves all the trans operations, and "de-activates" the transaction flag
         let ops = self.trans.commit_transaction().await?;
+
+        let _epoch = match ops.last() {
+            Some(DbRecord::Azks(azks)) => Ok(azks.latest_epoch),
+            other => Err(StorageError::Other(format!(
+                "The last record in the transaction log is NOT an Azks record {:?}",
+                other
+            ))),
+        }?;
```

Comment:
If this is solely to handle the error case, I think we can make this a bit more explicit with `if let` and avoiding the whole match + handling of `Some`. I think this same comment applies to any block where we're ignoring the result of the match.

Reply:
If we use an `if let` statement, we can only bind the result 1 time. There's no `if let Some(DbRecord::Azks(azks)) ..` and then still able to handle the result without using a `match` statement unfortunately.

Reply:
Yes, you're right! I misinterpreted this originally and now better realized the intent here. Thanks for clarifying!